### PR TITLE
Add example with multiple DID issuers from the same endpoint

### PR DESCRIPTION
**Pull request by OR13**
- Add did:key, did:elem, did:web, configuration for a single issuer

```
let body = {
    credential, 
    options: {
        proofPurpose: 'assertionMethod',
        issuer: assertionMethod.split('#')[0],
        verificationMethod: assertionMethod,
    } 
}
const vc = await fixtures.postJson('https://vc.transmute.world/vc-data-model/credentials', body);                    
```

Note the need to be able to specify the key used outside of the credential.

Technically, "issuer" is redundant.

### CODE REVIEW

File: test-suite/issuers/issuers-advanced.spec.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        describe(issuer, ()=>{
+            fixtures.issuerAssertionMethods[issuer].forEach((assertionMethod)=>{
+                describe(assertionMethod, ()=>{
+                    fixtures.credentials.forEach((credential)=>{
+                        it('should issue ' + credential.type[1], async ()=>{
+                            let body = {
+                                credential, 
+                                options: {
+                                    proofPurpose: 'assertionMethod',
+                                    issuer: assertionMethod.split('#')[0],
```
Comment (gannan08) [inline test-suite/issuers/issuers-advanced.spec.js]: This is okay for now. There may be an edge case where one can't extract the `issuer` via splitting on the `#` character.

File: test-suite/data/credential--vc.transmute.world--did:web:vc.transmute.world#z6MksHh7qHWvybLg5QTPPdG2DgEjjduBDArV9EF9mRiRzMBN--CertifiedMillTestReport.json [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+  "issuer": "did:web:vc.transmute.world",
+  "issuanceDate": "2020-03-09T18:19:10.033Z",
+  "credentialSubject": {
+    "id": "did:elem:EiAsRb0lUsZbTgobLKOGV-CesWOcHrp6RaRiIH7Vxvmu0g",
+    "cmtr": {
+      "companyName": "Steel Inc.",
+      "companyWebsite": "https://transmute.industries",
+      "companyAddress": "3260 46 Ave SE #30, Calgary, AB T2B 3K7, Canada",
+      "companyPhoneNumber": "555 555 5555",
+      "companyBrandMark": "https://www.freelogodesign.org/file/app/client/thumb/e2b4bee1-4228-4cf5-b6d7-148c4ad93a9b_200x200.png?1583363125286",
```
Comment (gannan08) [inline test-suite/data/credential--vc.transmute.world--did:web:vc.transmute.world#z6MksHh7qHWvybLg5QTPPdG2DgEjjduBDArV9EF9mRiRzMBN--CertifiedMillTestReport.json]: This comment should not hold up this PR. Hotlinking images may open up a privacy hole. It could be mitigated by using a trusted proxy to retrieve images on the client.

File: test-suite/issuers/issuers-advanced.spec.js (hunk 1 above)
Comment (OR13) [reply test-suite/issuers/issuers-advanced.spec.js]: yes, in such cases we would need to provide both... the main point here, is that we need a way to say who the issuer is AND a way to say which key to use... the former works with the current APIs, the latter is blocked by our lack of progress on support for "options" in the issuer api.

File: test-suite/data/credential--vc.transmute.world--did:web:vc.transmute.world#z6MksHh7qHWvybLg5QTPPdG2DgEjjduBDArV9EF9mRiRzMBN--CertifiedMillTestReport.json (hunk 2 above)
Comment (OR13) [reply test-suite/data/credential--vc.transmute.world--did:web:vc.transmute.world#z6MksHh7qHWvybLg5QTPPdG2DgEjjduBDArV9EF9mRiRzMBN--CertifiedMillTestReport.json]: This is a really excellent point.

**Comment by OR13** [comment]
@gannan08 because these changes are not supported by the current W3C standard APIs, I don't think we should merge this... The point of this PR was to motivate progress on those APIs, by demonstrating why what we have today is insufficient for greater interop.

**Comment by OR13** [comment]
I'm gonna close this, since the current APIs are not sufficient to support this for interop.
